portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsEntry.testcase

```diff
@@ -135,13 +135,42 @@ definition {
 	}
 
 	@description = "This is a test for LPS-136762. It checks that content can be added in full screen mode."
-	@ignore = "true"
 	@priority = "2"
 	test CanAddContentInFullScreenMode {
 		property portal.acceptance = "false";
 
-		// TODO LPS-136762 CanAddContentInFullScreenMode pending implementation
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Blogs Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "Blogs Page",
+			widgetName = "Blogs");
+
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "guest");
+
+		BlogsEntry.addTitle(entryTitle = "Blogs Entry Title");
+
+		BlogsEntry.switchContentViewMode(viewMode = "Fullscreen");
+
+		Type.typeAceEditor(
+			locator1 = "TextArea#ACE_EDITOR",
+			value1 = "Blogs Entry Content");
 
+		Button.clickDone();
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Liferay DXP");
```

Comment:
For the default site, usually, use the "Guest" as the parameter. Because the site name will be changed to different when the test runs into a different instance.